src/app.js

```diff
@@ -29,6 +29,7 @@ app.use(cookieSession({
 
   // Cookie Options. httpOnly is set by default to true for https
   sameSite: 'lax',
+  secureProxy: true,
```

Comment:
This looks great in dev, but seems to be breaking cookies in my local environment, preventing any logins when `BYPASS_AUTH` is false.

```suggestion
  secureProxy: (process.env.NODE_ENV === 'production'),
```